docs/trex_known_issues.md

```diff
@@ -17,6 +17,22 @@ For information about what is new or has changed in each release, see the [Relea
 Because of incompatibilities between Chrome and the internal Chromium-based browser used in Tableau, you can't use Chrome version 80 or later to debug your extensions. If you are using Tableau Desktop 2019.1 or later, you can debug extensions using Chrome version 79 or Chromium version 79. If you are using Tableau Desktop versions 2018.2 or 2018.3, you can use Chromium version 47. For more information about debugging extensions and using the Chromium browser, see [Debug Extensions in Tableau Desktop]({{site.baseurl}}/docs/trex_debugging.html) and [Download the Chromium Browser]({{site.baseurl}}/docs/trex_debugging.html#download-the-chromium-browser).
 
 
+### Unable to run dashboard extension on localhost or use self-signed certificates
+
+Tableau uses Qt WebEngine to embed browser pages within a dashboard. Qt WebEngine is built on Chromium. Because Tableau is now using a newer version of Qt WebEngine (5.15), and consequently, a newer version of Chromium, dashboard extensions running on localhost or with self-signed certificates (SSL) might not work in Tableau 2021.1, or in the most recent Tableau maintenance releases: 2020.2.7+, 2020.3.6+, and 2020.4.2+.
```

Comment:
1. What is the issue with localhost?
2. This seems internally focused with details that don't mean anything to external developers, like the vesion of QWebEngine we are using. How about this: The version of Chromium used to render extensions in Tableau Desktop has been updated. The newer version of Chromium includes some security enhancements. Consequently, dashboard extensions running on...

Reply:
@johnDance  
1. Good question.  I'm not sure what the issue is specifically with localhost. It was mentioned in the GitHub Issue ([370](https://github.com/tableau/extensions-api/issues/370)), but it might just be because they were trying to use a self-signed cert on localhost. 

2. Sure. Your suggestion looks good to me. On the other hand, is there an advantage in mentioning the specific versions of Qt and Chromium? I'm thinking that it gives someone more information if they need to track down issues. Or is this just asking for trouble, as Qt and Chromium are constantly releasing new versions.

Reply:
1. Okay, it was localhost with a self signed certificate. Not an issue with localhost itself. We can change the text to "localhost with a self-signed certificate". (drop the "or")

2. What does Qt WebEngine 5.15 tell anyone? How about this? "Tableau now uses Qt WebEngine 5.15, which is based on Chromium version 87.0.4280, with additional security fixes from newer versions." (I got that from the WebEngine docs.)

Reply:
I like it. Thanks. I found the same Chromium version number in their docs.
In the debugging topic, I was going to update a table with the corresponding Chrome, Chromium and Tableau versions. Should we list 87.0.4280 in the table? 

Tableau Desktop version  |  Chromium version  | Chrome version
|----|----|----|
2018.2, 2018.3 | 47.0.2526.0 |  Not available
2019.1 and later | 79.0.3945.0  | Chrome version 79 or earlier.
Latest maintenance release of 2020.2.7+, 2020.3.6+, 2020.4.2+ | 87.0.4280 | Chrome version 80 or later.
2021.1 and later | 87.0.4280 | Chrome version 80 or later.

Reply:
Yes. We should list it in the table.

Reply:
Also let's use "extensions running with self-signed certificates" and drop the localhost.

Reply:
Got it. Nix the localhost.